woocommerce/woocommerce-ios
# Order Details and Order Fulfillment screen layout changes for product list

Closes #1003.

Sorry about the extra LOC. It's because of all the changes I made to nib files.

Per the changes in Product Details i2, the Order Details > product list and Order Fulfillment ("pick list") are changing.

**i2 designs: Order Details > Product list**
<img width="395" alt="Screen Shot 2019-05-29 at 10 24 37 AM" src="https://user-images.githubusercontent.com/1062444/58569671-2f7d6700-81fc-11e9-8a21-bc32523afd20.png">

**i2 designs: Order Fulfillment ("pick list")**
<img width="384" alt="Screen Shot 2019-05-29 at 10 24 46 AM" src="https://user-images.githubusercontent.com/1062444/58569718-4754eb00-81fc-11e9-9a21-f8d8cb230424.png">

----

Proposed changes: each product should be its own cell, instead of one giant cell with multiple subviews. By creating individual cells, we can draw in the table in a more predictable fashion. Note that product images are not loaded here. That lives in a different task (#747).

Before | After
------------ | -------------
Current screen<br /><img src="https://user-images.githubusercontent.com/1062444/58571300-14602680-81ff-11e9-91f8-773827b042d2.png" width="300" /> | New layout with cells<br /><img src="https://user-images.githubusercontent.com/1062444/58571344-25109c80-81ff-11e9-8f0f-3a974a76a246.png" width="300" />
Current fulfillment button<br /><img src="https://user-images.githubusercontent.com/1062444/58571300-14602680-81ff-11e9-91f8-773827b042d2.png" width="300" /> | New fulfillment button in own cell<br /><img src="https://user-images.githubusercontent.com/1062444/58571380-3bb6f380-81ff-11e9-9e0f-efb4386434a9.png" width="300" />
Current fulfillment screen<br /><img src="https://user-images.githubusercontent.com/1062444/58571408-48d3e280-81ff-11e9-9067-994a5d4ad1f3.png" width="300" /> | New fulfillment screen with cells<br /><img src="https://user-images.githubusercontent.com/1062444/58571433-56896800-81ff-11e9-97c5-99999f0701a2.png" width="300" />

## To test
1. Navigate to Orders tab > Choose an order.
2. (Note the new cell layout)
3. Select the "fulfill order" button
4. On the fulfill order screen, note the new layout

Changes made to the code: 
- The product list in the Order Details VC no longer generates subviews and attaches them to a vertical uistackview in the product cell. Each product is now its own cell.
- The fulfill order button is now its own nib file and cell, instead of part of the uistackview in a single product cell.

Update release notes:
